playground/infrastructure/helper.py

```diff
@@ -30,9 +43,10 @@ class Example:
     code: str
     output: str
     status: STATUS_UNSPECIFIED
+    tag: Tag
 
 
-def find_examples(work_dir: str) -> List[Example]:
+def find_examples(work_dir: str, categories_path: str) -> List[Example]:
```

Comment:
I think it's better to get not the path but list of categories

Reply:
Changed.